pyaerocom/io/read_eea_aqerep_base.py

```diff
@@ -355,7 +355,11 @@ def read_file(self, filename, var_name, vars_as_series=False):
                     try:
                         data_dict[header[idx]][lineidx] = np.float64(rows[idx])
                     except (ValueError, IndexError):
-                        data_dict[header[idx]][lineidx] = np.nan
+                        try:
+                            data_dict[header[idx]][lineidx] = np.nan
+                        except Exception as e:
+                            print(f"FILENAME: {filename}")
```

Comment:
👍

Reply:
Not sure what this line has to do in this PR `matplotlib.contourf`?

Anyway, please don't use print but logger.error, including the exception.

Reply:
This was not supposed to be there but, removed from last commit

Reply:
but we learned today that it's good to log the filename so maybe it can still land in another PR

Reply:
I would like that, though it is low priority for a product which just has been discontinued...